fibonacci.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+function fibonacci(int $number)
+{
+	$array = [];
+	$array[0]=0;
+	$array[1]=1;
+	for ($i=2; $i<$number; $i++)
+	{
+		$array[$i] = $array[$i-1] + $array[$i-2];
+	}
+	
+	return $array;
+}
+
+
+function sum($number)
+{
+	$array=fibonacci($number);
+	$result=0;
+	
+		for ($i=0; $i<$number; $i++)
+	{
+		$result+=$array[$i];
+	}
```

Comment:
Так теж можна, але краще використовувати вбудовану функцію 
```suggestion
		$result = array_sum($array);
```